source/basic.tex

```diff
@@ -1106,7 +1106,7 @@
 \begin{codeblock}
 typedef unsigned char T;
 template<class T
-  = T               // lookup finds the typedef name of \tcode{\keyword{unsigned} \keyword{char}}
+  = T               // lookup finds the \grammarterm{typedef-name} of \tcode{\keyword{unsigned} \keyword{char}}
```

Comment:
We discussed removing the "of unsigned char" here but it's still present. Is that intentional?

Reply:
No. Removed for good.